specs/AppLifecycle/EnvironmentVariables/AppLifecycle - Environment Variables (EV).md

```diff
@@ -10,12 +10,12 @@ Environment variables is one example, where there are several existing APIs
 that provide partially overlapping functionality, and with some gaps. This spec
 describes the Reunion support for environment variables.
 
-Important Note: this feature relies on other platform features which are only
-available from Windows summer 2021 onwards. Therefore this feature is also 
-only available from Windows summer 2021 onwards. An app can check the 
-IsSupported property to determine whether the current device supports the 
+Important Note: The EV tracking feature relies on other platform features which are only
```

Comment:
'EV' isn't defined. Always define acronyms on first use

SUGGESTION: `The Environment Variables (EV) tracking feature...`

Reply:
It is defined in the title of the md.  Should I also define it in the document?

Reply:
> It is defined in the title of the md. Should I also define it in the document?

Lolz! I didn't even notice it in the filename! Though it's not in the title (line 1) so perhaps just add it there